redash/authentication/saml_auth.py

```diff
@@ -19,12 +21,26 @@ def get_saml_client(org):
 
     The configuration is a hash for use by saml2.config.Config
     """
-    metadata_url = org.get_setting("auth_saml_metadata_url")
+
+    metadata_inline_template = """<?xml version="1.0" encoding="UTF-8"?><md:EntityDescriptor entityID="{}" xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata"><md:IDPSSODescriptor WantAuthnRequestsSigned="false" protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol"><md:KeyDescriptor use="signing"><ds:KeyInfo xmlns:ds="http://www.w3.org/2000/09/xmldsig#"><ds:X509Data><ds:X509Certificate>{}</ds:X509Certificate></ds:X509Data></ds:KeyInfo></md:KeyDescriptor><md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="{}"/><md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="{}"/></md:IDPSSODescriptor></md:EntityDescriptor>"""
+
+    saml_type = org.get_setting("auth_saml_type")
     entity_id = org.get_setting("auth_saml_entity_id")
+    sso_url = org.get_setting("auth_saml_sso_url")
+    x509_cert = org.get_setting("auth_saml_x509_cert")
+    metadata_url = org.get_setting("auth_saml_metadata_url")
+
+    metadata_inline = metadata_inline_template.format(
+        entity_id, x509_cert, sso_url, sso_url
+    )
```

Comment:
Don't we need to do this only if "static" option enabled?

Reply:
Also instead of `format` let's use `mustache_render` from `redash.utils` (it's using the Mustache template langauge).